app/views/admin_mailer/notify_comment_moderators.html.erb

```diff
@@ -9,14 +9,21 @@
     https://<%= ActionMailer::Base.default_url_options[:host] %><%= @comment.node.path %>
   </p>
 
-  <p>
-    <a href="https://<%= ActionMailer::Base.default_url_options[:host] %>/admin/publish_comment/<%= @comment.id %>">Approve</a>
+    <p> <b> Comment: </b> </p>
+      <div style = "padding:2vw; background-color:rgb(217, 217, 217); border-radius:5px;">
+        <%= @comment.comment %>
+      </div>
+            
+  <p style="text-align:center; padding: 5px;">
+    <a href="https://<%= ActionMailer::Base.default_url_options[:host] %>/admin/publish_comment/<%= @comment.id %>"
```

Comment:
Please remove all the styles added so as to match other email templates. Thanks!